pkg/operator/cloudformation/error.go

```diff
@@ -0,0 +1,18 @@
+package cloudformation
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+)
+
+func isNoStateDriftError(err error) bool {
+	ioe, mat := err.(*types.InvalidOperationException)
+	if !mat {
+		return false
+	}
+
+	if ioe.ErrorCode() == "ValidationError" && ioe.ErrorMessage() == "No updates are to be performed." {
+		return true
+	}
```

Comment:
This is apparently how the AWS SDK forces us to match this specific error. Those error messages seem to be the "standard".